Userland/Libraries/LibGUI/Variant.h

```diff
@@ -135,9 +146,13 @@ class Variant {
             return as_bool() ? 1 : 0;
         if (is_float())
             return (int)as_float();
-        if (is_uint()) {
-            VERIFY(as_uint() <= INT32_MAX);
-            return (int)as_uint();
+        if (is_u32()) {
+            VERIFY(as_u32() <= INT32_MAX);
+            return (i32)as_u32();
+        }
+        if (is_u64()) {
+            VERIFY(as_u64() <= INT64_MAX);
+            return (i64)as_u32();
```

Comment:
```suggestion
            return (i64)as_u64();
```

Reply:
Okay, I just double checked the diff and this seems like the last typo :sweat_smile: